docker/docker-compose.yaml

```diff
@@ -0,0 +1,118 @@
+version: '3'
+
+services:
+  # cloudagent-db:
```

Comment:
I guess this is for later, but what is the story behind this db?

Reply:
Ah well spotted. I should probably clean that up. Basically the cloud-agent "should" be able to use a postgres backend which would be preferable when we look at mapping this to the cloud. For now though I'll clean this up and just use the local wallet